dts: gpio: Simplify the description of the binding

rruuaanng opened the pull request:
Remove redundant descriptions in GPIO bindings, such as "This is a representation of".

kartben commented on dts/bindings/gpio/andestech,atcgpio100.yaml:
```diff
@@ -1,8 +1,7 @@
 
-description:
-  This is a representation of Andes Technology atcgpio100 GPIO node
+description: Andes Technology atcgpio100 GPIO.
```
```suggestion
description: Andes Technology ATCGPIO100 GPIO Controller.
```

kartben commented on dts/bindings/gpio/rohm,bd8lb600fs-gpio.yaml:
```diff
@@ -6,7 +6,7 @@
 
 description: |
-    This is a representation of the Rohm BD8LB600FS SPI Gpio Expander.
+    Rohm BD8LB600FS SPI Gpio Expander.
```
```suggestion
    Rohm BD8LB600FS SPI GPIO Expander.
```

rruuaanng commented:
Wait, I found the missing change.

jerome-pouiller commented on dts/bindings/gpio/silabs,gecko-gpio-port.yaml:
```diff
@@ -1,4 +1,4 @@
-description: SiLabs Gecko GPIO port node
+description: SiLabs Gecko GPIO PORT
```
`port` is not an acronym. So, I am not sure we need to use uppercases.

rruuaanng replied:
To avoid any misunderstandings, I will modify them :)

jerome-pouiller commented on dts/bindings/gpio/ene,kb1200-gpio.yaml:
```diff
@@ -2,7 +2,7 @@
 
 description: |
-    ENE KB1200 GPIO(General purpose IO) Port node
+    ENE KB1200 GPIO(General purpose IO) Port
```
I assume we can get rid of `(General purpose IO)`.

rruuaanng committed:
dts: gpio: Simplify the description of the binding

Remove redundant descriptions in GPIO bindings, such
as "This is a representation of".

Signed-off-by: James Roy <rruuaanng@outlook.com>
